src/feed_handler.py

```diff
@@ -83,6 +92,27 @@ def _update_height(self, clob_pair_id: int, new_block_height: int):
                 f"Block height decreased from "
                 f"{self.heights[clob_pair_id]} to {new_block_height}"
             )
+
+    def _handle_subaccounts(self, update: subaccounts.StreamSubaccountUpdate):
+        """
+        Handle the StreamSubaccountUpdate message, updating the local subaccount state.
+        """
+        parsed_subaccount = subaccounts.parse_subaccounts(update)
+        subaccount_id = parsed_subaccount.subaccount_id
+
+        if update.snapshot:
+            # Replace the entire subaccount if this is a snapshot update
+            self.subaccounts[subaccount_id] = parsed_subaccount
+        else:
+            # Update the existing subaccount, or create a new one if it doesn't exist
```

Comment:
Can we simplify this logic? create if it doesn't exist, then one update flow. I think the example client should be as simple as possible

Reply:
done